CMakePresets.json

```diff
@@ -106,9 +106,13 @@
     },
     {
       "name": "release",
-      "displayName": "C++ Library",
-      "description": "C++ Library only",
-      "configurePreset": "release"
+      "displayName": "Release",
+      "description": "C++ Library and Python bindings",
+      "configurePreset": "release",
+      "targets": [
+        "neml2",
+        "python-stub"
+      ]
```

Comment:
These changes are already in main. Remember to rebase/merge before you merge this PR.